methods.go

```diff
@@ -296,6 +296,14 @@ func (b *Bicache) Resume() error {
 	return nil
 }
 
+// RemoveLeastRecentlyUsed remove the last element from
+// the MRU cache in each shard.
+func (b *Bicache) RemoveLeastRecentlyUsed() {
+	for _, s := range b.shards {
+		s.evictFromMRUTail(1)
```

Comment:
I don't recall enough of the code in the normal background shard accounting that calls this func, but it is wrapped in a lock and I suspect we would need a lock here as well.

Reply:
added locking https://github.com/jamiealquiza/bicache/pull/90/commits/3690125870b4e01bb26e8197891bd5dd8eabe0a7